src/components/cylc/tree/index.js

```diff
@@ -15,21 +15,22 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { extractGroupState } from '@/utils/tasks'
-import { computePercentProgress } from '@/components/cylc'
```

Comment:
The changes on this file are due to the move of the code to the `tree/deltas.js`, especially the code that was computing (really computing, as this was in a `computed` property of a component later :drum: ) the hierarchical tree, now in `CylcTree` and deltas module.